SU-HSDO/stanford_mrc
# MRCD8-236 mobile menu

# READY FOR REVIEW

# Summary
- Styles for mobile menu (600px and below; beginning at 599px)

# Needed By (Date)
- End of sprint (3/15)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that mobile menu is working and looks good
3. Note that the hover still appears on desktop machines; true test would be to inspect via mobile view on browser

# Affected Projects or Products
- DMRC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-236

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
